front/src/components/chips/CompanyChip.tsx

```diff
@@ -19,16 +20,33 @@ const StyledContainer = styled.span`
     filter: brightness(95%);
   }
 
+  [data-editmode='selected'] &:hover {
+    cursor: pointer;
+  }
+
+  [data-editmode='selected'] &:hover::after {
```

Comment:
it looks to me that the cross is not centered in the container on screenshot

Reply:
It's literally just a string, it _should_ be centered... but I agree, visually it looks like it's a bit off, I'll look into adding a margin.